### ISSUE TITLE
Playbook is confusing, please rename to e.g. runbooks

### ISSUE THREAD

**Issue by ericzolf** [redhat-cop/openshift-playbooks#208]
In the context of Ansible, a lot of people get confused by the term 'playbook', I would suggest to rename hence the concept of a project playbook to a project runbook to differentiate properly and avoid "stupid" questions.

**Comment by etsauer** [comment]
@ericzolf agreed. we're working on a full transition to a new brand and nomenclature.
